### ISSUE TITLE
Return Targets with the ExtractionBatch object on the extractionbatches endpoint

### ISSUE THREAD

**Issue by BlakeDraper** [USGS-WiM/liliservices#66]
The extractionbatch object within the Analysis Batch Detail endpoint's JSON response already has the targets array, so theoretically just a matter of applying that snippet to the extractionbatches endpoint as well. 

Possible issue though - it seems none of the ABs have any values in the Targets array of the extractionbatches array. This doesn't make sense since all the test extractionBatches I have created from the lili dev app have included targets (it's a requirement of extraction), so at some point in the logic, those targets (actually replicates) are either not being created and associated properly, or perhaps the target array is just not being assembled for the web service data response. 

So to recap: 1) determine why no target values are returning with the AB detail response and 2) please add the target array to the extractionbatches endpoint data reponse.
